crates/re_viewer/src/app.rs

```diff
@@ -1187,8 +1187,14 @@ fn about_rerun_ui(ui: &mut egui::Ui, build_info: &re_build_info::BuildInfo) {
         llvm_version
     };
 
+    let maybe_git_hash = if version.is_prerelease() {
```

Comment:
Maybe we should show this unconditionally? We are including a git commit for the rust version anyways so it's not like hiding this on releases actually makes for a cleaner user experience.

Reply:
Agreed